app/styles/app.scss

```diff
@@ -462,12 +462,6 @@ main {
     padding-top:1.1em;
 }
 
-
-#mfr {
-    padding-left: 25px;
```

Comment:
You can use osf-style helper for padding like this:
```
   ... class="p-l-lg p-r-lg"
```
https://centerforopenscience.github.io/osf-style/style.html#layout-helper